operator/v1/tests/consoles.operator.openshift.io/AAA_ungated.yaml

```diff
@@ -296,3 +296,138 @@ tests:
+          customization:
+            customLogos:
+              - type: Masthead
+                themes:
+                  - type: Default
+                    file:
+                      name: custom-logos-config
+                      key: masthead-default-logo.svg
+    - name: Should allow creating custom logos for the Default and any other theme type
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: Console
+        spec:
+          customization:
+            customLogos:
+              - type: Masthead
+                themes:
+                  - type: Default
+                    file:
+                      name: custom-logos-config
+                      key: masthead-default-logo.svg
+                  - type: Dark
+                    file:
+                      name: custom-logos-config
+                      key: masthead-dark-logo.svg        
+      expected: |
+        apiVersion: operator.openshift.io/v1
+        kind: Console
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+          customization:
+            customLogos:
+              - type: Masthead
+                themes:
+                  - type: Default
+                    file:
+                      name: custom-logos-config
+                      key: masthead-default-logo.svg
+                  - type: Dark
+                    file:
+                      name: custom-logos-config
+                      key: masthead-dark-logo.svg
+    - name: Should not allow duplicate theme types
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: Console
+        spec:
+          customization:
+            customLogos:
+              - type: Masthead
+                themes:
+                  - type: Dark
+                    file:
+                      name: custom-logos-config
+                      key: masthead-dark-logo.svg
+                  - type: Dark
+                    file:
+                      name: custom-logos-config
+                      key: masthead-dark-logo.svg
+      expectedError: "spec.customization.customLogos[0].themes[1]: Duplicate value: map[string]interface {}{\"type\":\"Dark\"}"
+    - name: Should not allow duplicate custom logo types
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: Console
+        spec:
+          customization:
+            customLogos:
+              - type: Masthead
+                themes:
+                  - type: Dark
+                    file:
+                      name: custom-logos-config
+                      key: masthead-dark-logo.svg
+              - type: Masthead
+                themes:
+                  - type: Light
+                    file:
+                      name: custom-logos-config
+                      key: masthead-light-logo.svg
+      expectedError: "spec.customization.customLogos[1]: Duplicate value: map[string]interface {}{\"type\":\"Masthead\"}"
```

Comment:
Nit, missing new line character at the end of the file. Makes blame inaccurate in the future